src/core/RFBStateProcessors/FrameBufferBellServerCutText.ts

```diff
@@ -13,7 +13,7 @@ class FrameBufferBellServerCutText implements IStateProcessor {
   wsSocket: ICommunicator
   next: IStateProcessor
   cmdSize: number
-  binaryEncDec: TypeConverter
+  // binaryEncDec: TypeConverter
```

Comment:
is this no longer needed?

Reply:
This variable is not being used anywhere. I have removed the commented out code